.github/workflows/python.yml

```diff
@@ -185,7 +185,7 @@ jobs:
         # This updates the cargo.lock file with the new version numbers and keeps the wheel build from failing
         run: |
           python3 scripts/version_util.py --patch_prerelease
-          cargo check
+          cargo update -w
```

Comment:
It is nice to use the full names (`--workspace`) in checked in files. It helps readability quite a lot.